scripts/lunch.coffee

```diff
@@ -93,12 +94,30 @@ module.exports = (robot) ->
 
   robot.respond /lunch locations$/i, (msg) ->
 
+    # sort by least latest visit then by least count
+    sort_lunch = (lunches) ->
```

Comment:
To match the style of this script, I'd move this to be a class method on `Lunch` (ie. `Lunch.sort(a, b)`)